### ISSUE TITLE
[Feature] Custom buttons (ribbon) rest call actions on selection of records

### ISSUE THREAD

**Issue by mennobeenes** [nocodb/nocodb#1041]
**Need for this feature**
Implementation of custom action on selection of table records.
For example, selecting 3 items in a table, click the button (in a ribbon).
Foreach selected item invoke a rest call with the current base, table name and id.

**Comment by spatical** [comment]
Definitely a feature I would use too.

**Comment by yureckey** [comment]
Yes, definitely something for custom action on a selected row is much needed - on the header/footer toolbar and/or as the buttons in the column. I'm surprised it's not implemented yet, it's kinda "must have" for dashboards.
Webhook for insert/update isn't cover all the cases (for example - when the user can't edit this table)

**Comment by hbouhadji** [comment]
+1

**Comment by CodeSugar** [comment]
+1

**Comment by DimiMikadze** [comment]
+1

**Comment by ManasMadrecha** [comment]
+1
